dependency-check-core/src/main/java/org/owasp/dependencycheck/analyzer/JarAnalyzer.java

```diff
@@ -747,6 +741,9 @@ protected boolean parseManifest(Dependency dependency, List<ClassNameInformation
                 foundSomething = true;
                 versionEvidence.addEvidence(source, "specification-version", specificationVersion, Confidence.HIGH);
             }
+        } catch (IOException ex) {
+            LOGGER.warn("Unable to read JarFile '{}'.", dependency.getActualFilePath());
```

Comment:
> Minor: maybe update the message to read something like `Unable to read dependency file '{}'.` as the {{JarAnalyzer}} can also handle {{war}} files.

Done in 6d7f7d8, thanks.

> I am not sure it is a good idea to swallow an `IOException` and log it at a Warn and Trace level.

The warning still gets displayed in our log e.g. `[WARN] Unable to read dependency file '...\__MACOSX\sample-application\._avro-ipc-1.5.0.jar'`. Also this is the same pattern as the other two usages of `try (JarFile jar = new JarFile(dependency.getActualFilePath()))`. What would be your preferred alternative?